src/tests/dotnet-stack/StackTests.cs

```diff
@@ -2,45 +2,71 @@
+  System.Console.il!System.ConsolePal+WindowsConsoleStream.Read(value class System.Span`1<unsigned int8>)
+  System.Console.il!System.IO.ConsoleStream.Read(unsigned int8[],int32,int32)
+  System.Private.CoreLib.il!System.IO.StreamReader.ReadBuffer()
+  System.Private.CoreLib.il!System.IO.StreamReader.Read()
+  System.Console.il!System.IO.SyncTextReader.Read()
+  System.Console.il!System.Console.Read()
+  ?!?";
+
+        private readonly string _correctStack60 = @"  [Native Frames]
+  System.Console.il!System.ConsolePal+WindowsConsoleStream.ReadFileNative(int,value class System.Span`1<unsigned int8>,bool,int32&,bool)
+  System.Console.il!System.ConsolePal+WindowsConsoleStream.Read(value class System.Span`1<unsigned int8>)
+  System.Console.il!System.IO.ConsoleStream.Read(unsigned int8[],int32,int32)
+  System.Private.CoreLib.il!System.IO.StreamReader.ReadBuffer()
+  System.Private.CoreLib.il!System.IO.StreamReader.Read()
+  System.Console.il!System.IO.SyncTextReader.Read()
+  System.Console.il!System.Console.Read()
+  StackTracee!Tracee.Program.Main(class System.String[])";
 
-        private readonly string correctStack = @"  [Native Frames]
-  System.Console!System.IO.StdInReader.ReadKey(bool&)
-  System.Console!System.IO.SyncTextReader.ReadKey(bool&)
-  System.Console!System.ConsolePal.ReadKey(bool)
-  System.Console!System.Console.ReadKey()
+        private readonly string _correctStack31 = @"  [Native Frames]
+  System.Console.il!System.ConsolePal+WindowsConsoleStream.ReadFileNative(int,unsigned int8[],int32,int32,bool,int32&,bool)
+  System.Console.il!System.ConsolePal+WindowsConsoleStream.Read(unsigned int8[],int32,int32)
+  System.Private.CoreLib.il!System.IO.StreamReader.ReadBuffer()
+  System.Private.CoreLib.il!System.IO.StreamReader.Read()
+  System.Console.il!System.IO.SyncTextReader.Read()
+  System.Console.il!System.Console.Read()
   StackTracee!Tracee.Program.Main(class System.String[])";
 
+        public static IEnumerable<object[]> Configurations => TestRunner.Configurations;
+
         public StackTests(ITestOutputHelper outputHelper)
         {
-            output = outputHelper;
+            _output = outputHelper;
         }
 
-        [Theory]
-        [InlineData("netcoreapp3.1")]
-        [InlineData("net5.0")]
-        public async Task ReportsStacksCorrectly(string traceeFramework)
+        [SkippableTheory, MemberData(nameof(Configurations))]
+        public async Task ReportsStacksCorrectly(TestConfiguration config)
         {
             Command reportCommand = ReportCommandHandler.ReportCommand();
 
             var console = new TestConsole();
             var parser = new Parser(reportCommand);
 
-            using TestRunner runner = new TestRunner(CommonHelper.GetTraceePathWithArgs(traceeName: "StackTracee", targetFramework: traceeFramework), output);
-            runner.Start();
+            await using TestRunner runner = await TestRunner.Create(config, _output, "StackTracee", usePipe: false);
```

Comment:
what does the left side await do? Is it for async disposal? Also, can the Create be called CreateAsync?

Reply:
Yes, TestRunner is IAsyncDisposable so the using needs to be await.  I don't like the "Async" prefix on Task functions and it might are convention when distinguishing sync from async  but I think it just makes things too verbose.  It is obvious if the function is async by the signature.